lib/jira/commands/transition.rb

```diff
@@ -58,14 +63,42 @@ def transitions
         @transitions ||= (
           transitions = {}
           metadata['transitions'].each do |transition|
-            transitions[transition['to']['name']] = transition['id']
+            transitions[transition['to']['name']] = {
+              id: transition['id'],
+              "fields"=> { "resolution"=> { "required"=> resolution_required?(transition) } }
```

Comment:
i dont think we need this to be a nested hash, we can just have it be
```
transitions[transition['to']['name']] = {
  id: transition['id'],
  resolution?: (
    transition['fields'].fetch('resolution', {}).fetch('required', false)
  )
}
```
or simply
```
transitions[transition['to']['name']] = {
  id:          transition['id'],
  resolution?: transition['fields']['resolution']['required'] rescue false
}
```